src/rabbit_ct_broker_helpers.erl

```diff
@@ -719,6 +726,25 @@ format_ipaddr_for_uri(
     Res1 = re:replace(Res0, "(^0(:0)+$|^(0:)+|(:0)+$)|:(0:)+", "::"),
     "[" ++ Res1 ++ "]".
 
+
+%% Virtual host management
+
+add_vhost(Config, VHost) ->
+    rabbit_ct_broker_helpers:rpc(Config, 0, rabbit_vhost, add, [VHost]).
+
+delete_vhost(Config, VHost) ->
+    rabbit_ct_broker_helpers:rpc(Config, 0, rabbit_vhost, delete, [VHost]).
```

Comment:
Nodes may not be clustered. As this is the responsability of the testcase, I would indicate the node in the test, not in `rabbit_ct_broker_helpers.erl`.